techniques/css/C42.html

```diff
@@ -181,20 +179,20 @@ <h4>The CSS</h4>
 
 
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
+				<h3>手順</h3>
 
-				<p>For each target that cannot be enlarged by a mechanism, is not inline, and is not covered by the essential exception:</p>
+				<p>メカニズムによって拡大できず、インラインでなく、必要不可欠の例外に該当しない各ターゲットに対して:</p>
 				<ul>
-					<li>Check that the target has enough spacing so that the space around it measures at least 44px width and 44px height.</li>
+					<li>ターゲットの周囲のスペースが少なくとも幅 44 px、高さ 44 px となるように、十分な間隔が確保されていることをチェックする。</li>
```

Comment:
Check that ... は、既存の他のテクニック (WCAG 2.1 達成方法集) では「〜を確認する」という訳になっています。

```suggestion
					<li>ターゲットの周囲のスペースが少なくとも幅 44 px、高さ 44 px となるように、十分な間隔が確保されていることを確認する。</li>
```